src/host_function.rs

```diff
@@ -53,11 +61,15 @@ impl HostFunctionList {
     }
 }
 
-pub fn pack_host_function(function_name: &CString, function_ptr: *mut c_void) -> NativeSymbol {
+pub fn pack_host_function(
+    function_name: &CString,
+    function_ptr: *mut c_void,
+    signature: CString,
+) -> NativeSymbol {
     NativeSymbol {
         symbol: function_name.as_ptr(),
         func_ptr: function_ptr,
-        signature: ptr::null(),
+        signature: signature.into_raw(),
```

Comment:
WAMR has changed register_host_function() signature in wasm_export.h recently. The 3rd arguments has been removed. therefore, I guess we shall remove `signature` field here.